packages/backend/src/nest/storage/storage.service.ts

```diff
@@ -419,6 +423,25 @@ export class StorageService extends EventEmitter {
     this.logger('STORAGE: Finished creating certificatesRequests db')
   }
 
+  public async filterCsrs(allCsrs: string[]): Promise<string[]> {
```

Comment:
I would probably make this "getCsrs" and include
```
const allCsrs = this.getAllEventLogEntries(this.certificatesRequests)
```
inside the function, since I think the only time we want to retrieve CSRs is if they are filtered.